.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
+  schedule:
+    - cron:  '0 3 * * *' # daily, at 3am
+
+jobs:
+  #lint:
+  #  name: Linting
+  #  runs-on: ubuntu-latest
+  #
+  #  steps:
+  #    - uses: actions/checkout@v1
+  #    - uses: actions/setup-node@v1
+  #      with:
+  #        node-version: 10.x
+  #
+  #    - name: get yarn cache dir
+  #      id: yarn-cache
+  #      run: echo "::set-output name=dir::$(yarn cache dir)"
+  #
+  #    - uses: actions/cache@v1
+  #      with:
+  #        path: ${{ steps.yarn-cache.outputs.dir }}
+  #        key: ${{ runner.os }}-yarn-lint-${{ hashFiles('**/yarn.lock') }}
+  #        restore-keys: |
+  #          ${{ runner.os }}-yarn-
+  #
+  #    - name: install dependencies
+  #      run: yarn install
+  #
+  #    - name: lint
+  #      run: yarn lint
```

Comment:
Linting is currently not running with the old Travis setup neither – we'll re-enable (and migrate away from `eslint-config-simplabs`) in a second step.

Reply:
then I'd prefer to not keep it in the file, or at least add that as a comment to the file

Reply:
ok, sure – let's keep it out for now